src/Felix/Instances/Setoid/Type.agda

```diff
@@ -0,0 +1,52 @@
+{-# OPTIONS --safe --without-K #-}
+
+module Felix.Instances.Setoid.Type where
+
+open import Data.Empty.Polymorphic renaming (⊥ to ⊥′)
+open import Data.Product.Relation.Binary.Pointwise.NonDependent using (×-setoid)
+open import Data.Sum.Relation.Binary.Pointwise using (⊎-setoid)
+open import Data.Unit.Polymorphic renaming (⊤ to ⊤′)
+open import Function using (Func)
+open import Function.Construct.Setoid renaming (setoid to ⟶-setoid)
+open import Level using (Level; _⊔_)
+open import Relation.Binary using (Setoid)
+
+private
+  variable
+    a b ℓ₁ ℓ₂ : Level
+
+infixr 0 _⟶_
+_⟶_ : Setoid a ℓ₁ → Setoid b ℓ₂ → Set (a ⊔ b ⊔ ℓ₁ ⊔ ℓ₂)
+_⟶_ = Func
+
+open Func public
```

Comment:
I don't think I'd `open` publicly.

Reply:
It exposed just `to` and `cong`. I've limited to `cong` given that we have an operator form for `to`. Unless `open`ing `public`ly does something else that I am not aware of?

Reply:
I think `public` here means that the all of the `Func` fields & methods get exported from `Felix.Instances.Setoid.Type` and thus from `Felix.Instances.Setoid`, including `to`, `cong`, `isCongruent`, `Eq₁`, and `Eq₂`.

Reply:
Oh, you're completely right, forgot about other declarations. I've limited the `cong` anyway